corehq/apps/accounting/tests/test_domain_user_history.py

```diff
@@ -51,6 +56,60 @@ def test_calculate_users_in_all_domains(self):
         self.assertEqual(domain_user_history.record_date, self.record_date)
 
 
+@es_test(requires=[form_adapter], setup_class=True)
+class TestFormSubmittingMobileWorkerHistory(BaseInvoiceTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        num_workers = 5
+        generator.arbitrary_commcare_users_for_domain(cls.domain.name, num_workers)
+
+        cls.num_form_submitting_workers = 3
+        cls.one_day_ago = datetime.date.today() - datetime.timedelta(days=1)
+        one_week_ago_dt = datetime.datetime.combine(
+            cls.one_day_ago - datetime.timedelta(days=6), datetime.time()
+        )
+        for user in cls.domain.all_users()[:cls.num_form_submitting_workers]:
+            cls._submit_form(user, one_week_ago_dt)
+
+    @classmethod
+    def _submit_form(cls, user, received_on):
+        form_pair = make_es_ready_form(
+            TestFormMetadata(
+                domain=cls.domain.name,
+                user_id=user.user_id,
+                received_on=received_on
+            )
+        )
+        form_adapter.index(form_pair.json_form, refresh=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        delete_all_users()
+        delete_all_domains()
+        return super().tearDownClass()
+
+    def test_form_submitting_mobile_worker_history(self):
+        worker_history = FormSubmittingMobileWorkerHistory.objects.create(
+            domain=self.domain.name,
+            num_users=self.num_form_submitting_workers,
+            record_date=self.one_day_ago
+        )
+        self.assertEqual(worker_history.domain, self.domain.name)
+        self.assertEqual(worker_history.num_users, self.num_form_submitting_workers)
+        self.assertEqual(worker_history.record_date, self.one_day_ago)
+
+    def test_calculate_form_submitting_mobile_workers_in_all_domains(self):
+        tasks.calculate_form_submitting_mobile_workers_in_all_domains()
+        self.assertEqual(FormSubmittingMobileWorkerHistory.objects.count(), 1)
+
+        worker_history = FormSubmittingMobileWorkerHistory.objects.first()
+        self.assertEqual(worker_history.domain, self.domain.name)
+        self.assertEqual(worker_history.num_users, self.num_form_submitting_workers)
+        self.assertEqual(worker_history.record_date, self.one_day_ago)
+
```

Comment:
> what do you think of moving it to a `test_tasks.py` module instead?

I think that makes sense here. Since we can trust that Django creates an object using the properties we give it, what's left to be tested here is really just the task that's used to generate the `FormSubmittingMobileWorkerHistory`.